tests/testthat/test-fb_count_traits_by_species.R

```diff
@@ -0,0 +1,57 @@
+# Fake dataset ----
```

Comment:
Same comments as for fb_count_species_by_traits():

- [ ] Good inputs should be in a separate `test_that()` call
- [ ] What happens when a trait has only NAs?
- [ ] What happens when a species has only NAs?

Reply:
Respectively done in 3bacaf7, xxx, xxx